### DISCUSSION TITLE
Doubt about validation

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#493] [Q&A] [answered]
Good Morning Lab !
I've got a doubt about validation.
I've just sent twice a notice type 11, before sending it I always call validation url (https://cvs.preview.ted.europa.eu/v1/notices/validation) and the notice is valid 
But the result of publication (https://enotices2.preview.ted.europa.eu/esenders/api/v2/notice/notices/submit ) was a Schematron file with the error : 
**********************************
id :
location : /pin:PriorInformationNotice/cbc:IssueDate
failedAssertText : Notice Dispatch Date must be between 0 and 24 hours before the current date.

![image](https://github.com/OP-TED/eForms-SDK/assets/52211037/2f5859fd-404a-4639-9c88-d656cb9f9433)

I do understand the error, what I do not understand is why I did not get this error by validation.
Are there 2 different validation ?
Thanks and regards
Renato

**Comment by bertrand-lorentz** [comment, marked as answer]
The validation API has 2 validation mode : "static" and "dynamic".
See the docs for the corresponding parameter at https://docs.ted.europa.eu/api/endpoints/cvs-ted-europa-eu.html#InputNoticeValidation

The default mode is static, but eNotices2 uses the dynamic mode when a notice is submitted for publication.

The "dynamic" mode has all the validation rules from the "static" mode, plus rule that depend on information outside of the notice being validated. As indicated in the [page for the "Preview" environment](https://docs.ted.europa.eu/home/eforms/preview/index.html#_ted_central_validation_service), currently the only additional rules in "dynamic" is the check on the dispatch date (BT-05) (which corresponds to the error you got). For this rule the information outside of the notice is the current date.
